tests/__fixtures__/testWithTranslationHOC/classComponent.js

```diff
@@ -2,28 +2,40 @@ import * as React from 'react';
 import { withTranslation } from 'react-i18next';
 import * as ReactI18Next from 'react-i18next';
 
+function ensureT(message, t) {
+  if (!t) {
+    console.log(message);
+  }
+}
+
+
 class MyComponent0 extends React.PureComponent {
   render() {
+    ensure('not-extracted-0', this.props.t);
```

Comment:
I suggest that you don't change this file and just create a new test file that tests this bug specifically.

Reply:
You're right. Done.